lib/powertrack/rules/rule.rb

```diff
@@ -0,0 +1,93 @@
+require 'multi_json'
+
+module PowerTrack
```

Comment:
I don't feel like there's a guideline. Ruby, unlike Java, does not enforce an automatic module to directory mapping. Here I use the rules directory to include everything related to rules. More to come BTW.

But I still want the Rule class to be available under the PowerTrack module.